fem/datacollection.cpp

```diff
@@ -598,14 +605,28 @@ void VisItDataCollection::LoadFields()
       // TODO: 1) load parallel GridFunction on one processor
       if (serial)
       {
-         field_map.Register(it->first, new GridFunction(mesh, file), own_data);
+         if ((it->second).association == "nodes")
```

Comment:
What are the valid values for ``VisItFieldInfo.association``?
Looking at ``RegisterField`` and ``RegisterQField``, this looks right.

Perhaps some comments can be added here?
https://github.com/mfem/mfem/blob/d87f4e347c1332181263a8f90adb6e66db479d01/fem/datacollection.hpp#L388-L397

Reply:
In Visit there is already a hook for the `elements` instead of `qpoints` I introduced here.
So I think it might be better to adopt `elements`, which is currently not used, as far as I know.

Reply:
There is also the issue of the LOD. I would like to use that to tell Visit the integration order.
Or should I introduce a new parameter in the file to  pass this info to visit.
Then there is also the issue of mapping an integration routine with the appropriate refinement routine in visit. I am aiming for a minimal implementation with abundant checks for cases that will not work.